### PR TITLE
Update host doco

### PR DESCRIPTION
**Pull request by weralabaj**
Connects to https://github.com/Particular/NServiceBus.Host/issues/68

### CODE REVIEW

**Comment by weralabaj** [comment]
@bording @SeanFeldman Have a look to, please.

File: nservicebus/upgrades/5to6removePShelpers.include.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<configuration>
+  <configSections>
+    <section name="AuditConfig" 
+             type="NServiceBus.Config.AuditConfig, NServiceBus.Core" />
+  </configSections>
+  <AuditConfig QueueName="audit" />
+</configuration>
+```
+
+In Version 6 and higher these helpers have been removed. The configuration helpers encouraged creating a more complex xml configuration than was necessary, making it difficult to manage in the long-run.
```
Comment (SeanFeldman) [inline nservicebus/upgrades/5to6removePShelpers.include.md]: xml should be capitalized?

File: nservicebus/upgrades/5to6removePShelpers.include.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <section name="AuditConfig" 
+             type="NServiceBus.Config.AuditConfig, NServiceBus.Core" />
+  </configSections>
+  <AuditConfig QueueName="audit" />
+</configuration>
+```
+
+In Version 6 and higher these helpers have been removed. The configuration helpers encouraged creating a more complex xml configuration than was necessary, making it difficult to manage in the long-run. 
+
+The recommended configuration approach is "code-first", which is more flexible and less error prone than using Powershell helpers. The configuration can be read from any location at runtime, including the `app.config`. For more information about the API refer to the configuration documentation dedicated to the particular functionality. For example, see how [Error](/nservicebus/errors/#configure-your-error-queue-using-code) and  [Audit](/nservicebus/operations/auditing.md#configuring-auditing-using-code) queues can be configured using code API.
```
Comment (SeanFeldman) [inline nservicebus/upgrades/5to6removePShelpers.include.md]: "error prone" -> "error-prone"

**Comment by bording** [comment]
I took a look at the changes and made some wording tweaks.

I've also rebased the branch to deal with the merge conflicts introduced by the beta1 core update.

**Pull request merged**